test/backend/inference_backend.py

```diff
@@ -698,7 +698,8 @@ def get_test_models():
         # ==MIN== 1
         # ==LIM== Does not support int4 and uint4.
         # By def, no dynamic shapes.
-        "test_constant_cpu": {STATIC_SHAPE: {}},
+        # Commented out because the model has no input but one is provided
+        # "test_constant_cpu": {STATIC_SHAPE: {}},
```

Comment:
This should work now since we can handle functions with no inputs.

Reply:
Yes, I reversed the code but forgot the comment.